classes/local/attributemapping_list.php

```diff
@@ -81,6 +81,10 @@ public function get_text_content() {
     private function validate_attributemapping($attributemappings) {
         $uniqueattributes = [];
         foreach ($attributemappings as $attributemapping) {
+            if (!$attributemapping->accredibleattribute) {
+                continue;
+            }
```

Comment:
Excluding non-selected values from uniqueness validation.

Reply:
This should be reverted once multiple selection is implemented.